x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_flyout_provider.tsx

```diff
@@ -22,7 +22,7 @@ import { MLFieldStatsFlyoutContext } from './use_field_stats_flytout_context';
 import { PopulatedFieldsCacheManager } from './populated_fields/populated_fields_cache_manager';
 
 export const FieldStatsFlyoutProvider: FC<{
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
This was causing an error in `x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_flyout_provider.tsx` if left as a required field